src/video/windows/SDL_windowsevents.c

```diff
@@ -1195,6 +1198,8 @@ LRESULT CALLBACK WIN_WindowProc(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lPara
 
     case WM_POINTERDOWN:
     case WM_POINTERUP: {
+        returnCode = 0;  // always mark this event handled, so it doesn't go to DefWindowProc (see #11843).
```

Comment:
It's likely that always handling these messages might regress touch input. Since SDL is lying to windows about handling the PT_TOUCH messages, even though it's not doing anything with them (i.e. not sending to the application).

The two issues I opened show violation of the documentation, but any real-world impact is untested. I invite you to you to test some of the hypothetical problems I listed. I'm currently travelling and don't have access to my touchscreen monitor and graphics tablet to test. Sorry if my original issue description is unclear about this.

Reply:
Yeah, I might back this part out for now and wait for bug reports to arrive...I'm not really clear that this is even what Microsoft is asking me to do, honestly.

Reply:
Microsoft likely wants you to handle all pointer events properly, but SDL already has perfectly good touch handling via `WM_TOUCH`. I'm fine with only the `WM_POINTERCAPTURECHANGED` change going in (please update the PR title and description accordingly). I'll try to borrow some graphics tablets and touchscreen monitors and do some testing as set out in https://github.com/libsdl-org/SDL/issues/11843.